salt/knox/init.sls

```diff
@@ -26,6 +23,23 @@
 {% set opentsdb_version = pillar['opentsdb']['version'] %}
 {% set helper_directory = knox_home_directory + '/helper' %}
 
+{%- if grains['hadoop.distro'] == 'CDH' -%}
```

Comment:
Please see above comment about Hadoop usage and avoid replication like this, it leads to bugs (see DRY)